### PR TITLE
Organization of  ConsoleCommands/Messages

### PR DESCRIPTION
**Pull request by Wojdylak**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.13
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | yes
| Related tickets | N/A
| License         | MIT

All CLI commands → Console/Command
All Message → Command

List of not released clases: 
- \Sylius\Bundle\AdminBundle\Command\Factory\QuestionFactory
- \Sylius\Bundle\AdminBundle\Command\Factory\QuestionFactoryInterface
- \Sylius\Bundle\AdminBundle\Command\ChangeAdminUserPasswordCommand
- \Sylius\Bundle\AdminBundle\Command\CreateAdminUserCommand

Due to validation issues (e.g., in `src/Sylius/Bundle/ApiBundle/Resources/config/validation/AdminResetPassword.xml`) that do not recognize aliases created on the class, I am unable to move classes from the `Message`, `MessageDispatcher`, and `MessageHandler` folders. I have only added information in the upgrade file that this change will be made in version 2.0.

### CODE REVIEW

File: src/Sylius/Bundle/CoreBundle/Console/Command/InformAboutGUSCommand.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                'For purely statistical purposes and in order to inform you about important updates and security patches, Sylius might send non-sensitive data to our servers. We send:',
+                '* Hostname',
+                '* User-agent',
+                '* Locale',
+                '* Environment (test, dev or prod)',
+                '* Currently used Sylius version',
+                '* Date of the last contact',
+                'If you do not consent please follow this cookbook article:',
+                'https://docs.sylius.com/en/latest/cookbook/configuration/disabling-admin-notifications.html',
+                'That being said, every time we get a notification about a new site deployed with Sylius, it brings a huge smile to our face and motivates us to continue our Open Source work.',
```
Comment (NoResponseMate) [inline src/Sylius/Bundle/CoreBundle/Console/Command/InformAboutGUSCommand.php]:
```suggestion
                'That being said, every time we get a notification about a new site deployed with Sylius, it brings a huge smile to our faces and motivates us to continue our Open Source work.',
```
lol

**Review by NoResponseMate — DISMISSED** [review]

**Review by jakubtobiasz — DISMISSED** [review]

**Comment by jakubtobiasz** [comment]
I resolved the conflicts 🖖🏻.

File: src/Sylius/Bundle/UserBundle/Resources/config/services.xml [hunk 2]
```diff
@@ -27,14 +27,18 @@
     <services>
         <defaults public="true" />
 
-        <service id="Sylius\Bundle\UserBundle\Command\DemoteUserCommand">
+        <service id="Sylius\Bundle\UserBundle\Console\Command\DemoteUserCommand">
             <tag name="console.command" />
         </service>
 
-        <service id="Sylius\Bundle\UserBundle\Command\PromoteUserCommand">
+        <service id="Sylius\Bundle\UserBundle\Command\DemoteUserCommand" alias="Sylius\Bundle\UserBundle\Console\Command\DemoteUserCommand" />
```
Comment (diimpp) [inline src/Sylius/Bundle/UserBundle/Resources/config/services.xml:34]: What do you think adding deprecations to the aliases as well?

https://symfony.com/doc/current/service_container/alias_private.html#deprecating-service-aliases

File: src/Sylius/Bundle/AdminBundle/Tests/Console/Command/CreateAdminUserCommandTest.php [hunk 3]
```diff
@@ -11,14 +11,14 @@
...
-namespace Sylius\Bundle\AdminBundle\Tests\Command;
+namespace Sylius\Bundle\AdminBundle\Tests\Console\Command;
 
 use PHPUnit\Framework\MockObject\MockObject;
 use PHPUnit\Framework\TestCase;
-use Sylius\Bundle\AdminBundle\Command\CreateAdminUserCommand;
-use Sylius\Bundle\AdminBundle\Command\Factory\QuestionFactoryInterface;
-use Sylius\Bundle\AdminBundle\Exception\CreateAdminUserFailedException;
 use Sylius\Bundle\AdminBundle\Message\CreateAdminUser;
+use Sylius\Bundle\AdminBundle\Console\Command\Factory\QuestionFactoryInterface;
```
Comment (diimpp) [inline src/Sylius/Bundle/AdminBundle/Tests/Console/Command/CreateAdminUserCommandTest.php:18]: Sorting wise, those `use`s were in correct place.

File: src/Sylius/Bundle/UserBundle/Resources/config/services.xml (hunk 2 above)
Comment (Wojdylak) [reply src/Sylius/Bundle/UserBundle/Resources/config/services.xml:34]: The files contain information about deprecation, so I think it is enough.

**Review by jakubtobiasz — APPROVED** [review]

**Review by NoResponseMate — APPROVED** [review]

**Comment by NoResponseMate** [comment]
Thank you, @Wojdylak!

**Pull request merged**